google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -383,7 +383,7 @@
 		"setStatement": {
 			"propertyName": "RPC_PRIORITY",
 			"separator": "=",
-			"allowedValues": "'(HIGH|MEDIUM|LOW)'",
+			"allowedValues": "'(HIGH|MEDIUM|LOW|NULL)'",
```

Comment:
Adding a new value to `allowedValues` means that you also need to regenerate the tests so it will also include tests with the new value:

`mvn -P generate-test-sql-scripts compile`